src/collisions_updater.cpp

```diff
@@ -0,0 +1,228 @@
+#include <ros/ros.h>
+#include <ros/package.h> // for getting file path for loadng images
+
+#include <moveit/setup_assistant/tools/compute_default_collisions.h>
+#include <moveit/setup_assistant/tools/moveit_config_data.h>
+
+#include <boost/filesystem.hpp>  // for reading folders/files
+#include <boost/program_options.hpp>
+#include <boost/algorithm/string/join.hpp>
+
+#include <fstream>  // for reading in urdf
+#include <streambuf>
+
+namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+
+bool loadFileToString(std::string& buffer, const std::string& path, const std::vector<std::string> &xacro_args){
+
+  if(path.find(".xacro") != std::string::npos){
+        std::string cmd = "rosrun xacro xacro ";
```

Comment:
The indentation here is too much. In general (and as the original author of the setup assistant) I think we should continue using the ROS style guidelines such as not having whole functions defined on one line, e.g. not:

> bool operator < (const SortableDisabledCollision &other) const { return key_ < other.key_; }

and having curly braces be line breaking, e.g. not:

> if( config_data.urdf_pkg_name_.empty() || config_data.urdf_pkg_name_ == "\"\"" ){

But maybe as a whole people don't care enough? I use clang-format to auto format all my code to ROS guidelines